ubuntu/common/check_nvidia_uvm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+#Catch error codes that may be thrown by the executable passed as the first
+#input, and if an error code is tripped throw the second input as a message
+catch_error() {
+	output=$($1)
+	err_code=$?
+	if [ $err_code -ne 0 ]; then
+		if [ $3 -eq 1 ]; then
+			echo "$2 $err_code" >&2
+		fi
+		exit $err_code;
+	fi
+}
+
+catch_error_grep() {
+	output=$1
+	err_code=$?
+	if [ $err_code -ne 0 ]; then
+		if [ $3 -eq 1 ]; then
+			echo "$2 $err_code" >&2
+		fi
+		exit $err_code;
+	fi
+}
+
+if test -f "/dev/nvidia-uvm"; then
+        exit 0;
+else
+        :
+fi
+
+verb=0
+while [ $# -gt 0 ] ; do
+	if [ $1 = "-v" ]; then
+		verb=1
+	fi
+	shift
+done
```

Comment:
Not sure if we really need this? Could you point me where this script is invoked with -v option?

Reply:
We don't need it for now, but I just added it so we can run the script and have the error codes printed out if we need to debug something. I can get rid of it if you think it's confusing?

Reply:
I see. Since we are not actually invoking this script with -v option somewhere, I prefer we remove it.

Reply:
Ok, removed.